scripts/convert.py

```diff
@@ -26,6 +26,7 @@ def main(args_in: Optional[List[str]] = None) -> None:
         default="f32",
     )
     parser.add_argument("--outfile", type=Path, required=True, help="path to write to")
+    parser.add_argument("--whisper_repo_path", type=Path, required=False, help="path to whisper repo")
```

Comment:
what kinds of path, can you clarify on readme?

Reply:
I will rm it.